TelegramBots.wiki/FAQ.md

```diff
@@ -282,3 +283,24 @@ After that your bot will look like:
   }
 ```
 Also you could just implement LongPollingBot or WebHookBot interfaces. All this bots will be registered in context and connected to Telegram api.
+
+
+## <a id="list_of_group_members"></a>How to get a list of all group members ##
+
+Telegram Bot API does not allow you to get a list of all group members.
+However, you can create your own list of users:
+```java
+   public class YourBotClassName extends TelegramLongPollingBot {
+     List<User> groupMembers = new ArrayList<>();
+     //Bot body.
+   }
+```
+And if someone joins the group, add a new user to the list:
+```java
+   @Override
+   public void onUpdateReceived(Update update) {
+     if (update.getMessage().getNewChatMembers() != null) {
```

Comment:
getNewChatMember always returns something. If newChatMembers is null it will return an empty list.

https://github.com/rubenlagus/TelegramBots/blob/8351707e835bea544aa183ead179bb2fda0788d2/telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/Message.java#L302